### ISSUE TITLE
Why the 'bias' is set to False?

### ISSUE THREAD

**Issue by shuuchen** [flyyufelix/DenseNet-Keras#9]
Hi,

Is it possible to get better results by setting 'bias' to false?

**Comment by flyyufelix** [comment]
The 'bias' term is encapsulated in the BatchNorm layer. Notice that 'bias' term is dropped for ResNet too
